src/components/Person/Person.jsx

```diff
@@ -1 +1,15 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">My name is {person.name}</h2>
+    {person.age ? <p className="Person__age">I am {person.age}</p> : ''}
+    {person.isMarried ? (
+      <p className="Person__partner">
+        {person.sex === 'f'
+          ? `${person.partnerName} is my husband`
+          : `${person.partnerName} is my wife`}
+      </p>
+    ) : (
+      <p className="Person__partner">I am not married</p>
+    )}
```

Comment:
Preferalterfing tag content over the whole tag

```suggestion
   <p className="Person__partner">
      {person.isMarried ? `${person.partnerName} is my ${person.sex === 'm' ? 'wife' : 'husband'}` : 'I am not married'
   </p>
```